examples/tactics/eci19/ConstructiveLogic.fst

```diff
@@ -14,13 +14,14 @@ commonly, however, the programmer needs to give the SMT solver some
 help in the form of new facts that it can use. We've seen examples of
 this when we call lemmas within a function body in order to make it
 verify. For instance, compare the following two examples (the first of
-which fails).
+which in the comments is known to be unstable).
 *)
 
-[@expect_failure]
+(*
```

Comment:
This is interesting. The example that was showing failing smt proofs, now works.

Reply:
Weird! I noticed the same thing in a completely separate branch, so maybe some dependency got updated? In fact even master does not pass (local) ci for me right now.

Reply:
There seems to be a difference between running with --query_stats and without. CI will pass using that flag (this definition will fail as expected). You might need to revert this change to get a green. I'll look to see what causes this.

Reply:
Solved in #3010